### ISSUE TITLE
Possible retina screen issue, experiencing pixelated UI

### ISSUE THREAD

**Issue by BarryMode** [Bionus/imgbrd-grabber#1453]
**Bug description**

Pixelated UI

**Steps to reproduce**

1. Build on Mac
2. Open program

**Expected behavior**

Normal UI

**Screenshots**

<img width="1035" alt="screen shot 2018-10-03 at 8 02 35 pm" src="https://user-images.githubusercontent.com/5648875/46448053-b4d27980-c74a-11e8-9ad5-a167fac61231.png">

**System information**

- OS: [e.g. MacOS Mojave]
- Grabber version: [e.g. 6.0.6]

**Additional context**

It's possible this issue may be unique to me like maybe I'm building it wrong or something like that. The screenshot should say it all. If there's anything I can do to help, let me know. I did some initial searching on this issue regarding QT5 and retina. I tried a couple of things and didn't have any luck.

**Comment by Bionus** [comment]
> I did some initial searching on this issue regarding QT5 and retina. I tried a couple of things and didn't have any luck.

Have you checked [High DPI Displays](https://doc.qt.io/qt-5/highdpi.html) and [High-DPI Support in Qt 5.6](https://blog.qt.io/blog/2016/01/26/high-dpi-support-in-qt-5-6/)?

A few options that I cannot personally try:

1. Make sure to build with the latest Qt version
2. Put this in the app plist file:
```xml
<key>NSPrincipalClass</key>
<string>NSApplication</string>
<key>NSHighResolutionCapable</key>
<string>True</string>
```
3. Put this in [gui/src/main/main.cpp](https://github.com/Bionus/imgbrd-grabber/blob/master/gui/src/main/main.cpp#L50) (at the very beginning of the method, **before** the `QApplication` instantiation), then re-build:
```cpp
QGuiApplication::setAttribute(Qt::AA_EnableHighDpiScaling);
```
4. Set `QT_DEVICE_PIXEL_RATIO=2` before running grabber (either in your env or local to the Grabber run)

Try those one by one and if any work I'll commit a fix directly in the code base 👍

**Comment by BarryMode** [comment]
I'm not sure what the deal was, but I can't even test it now because the problem isn't happening anymore. Weird right? I did the plist thing and then opened the program and it looked normal, so I thought that fixed it, however, when I removed the plist file, it still looked normal.

If it happens again, I'll open this back up and follow more of your steps. I'm pretty sure I had never seen this happen before.
